### ISSUE TITLE
[BUG] in GET /v2/entiies using "q" query parameter, not returns error if left-hand-side missing

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#1754]
in GET /v2/entiies using "q" query parameter, not returns error if left-hand-side missing.

#### Create Request
```
POST http://localhost:1026/v2/entities
Content-Type: application/json
Fiware-Service: test_list_only_q
Fiware-ServicePath: /test
 {"type": "vehicle", "speed": {"type": "km_h", "value": "89", "metadata": {"very_hot_1": {"type": "alarm", "value": "LQJI1NQVro"}, "very_hot_0": {"type": "alarm", "value": "LQJI1NQVro"}}}, "id": "car_3"}
```

#### GET request
```
GET http://localhost:1026/v2/entities?q===speed
Content-Type: application/json
Fiware-Service: test_list_only_q
Fiware-ServicePath: /test
```

#### GET response
```
http code: 200
date: Wed, 03 Feb 2016 10:52:29 GMT
content-length: 2
content-type: application/json
 []
```

#### expected response 
```
http code: 400
date: Wed, 03 Feb 2016 11:02:39 GMT
content-length: 63
content-type: application/json
{
   "error":"BadRequest",
   "description":"invalid query expression"
}
```

**Comment by iariasleon** [comment]
LGTM
```
http code: 400 - Bad Request
headers:
   date: Tue, 19 Jul 2016 09:19:27 GMT
   fiware-correlator: e4263b02-4d91-11e6-852c-005056a20feb
   connection: Keep-Alive
   content-type: application/json
   content-length: 69
payload: {"error":"BadRequest","description":"empty left-hand-side in q-item"}

```

**Comment by fgalan** [comment]
Looking to release notes, it seems this issue was solved in 0.28.0. See https://github.com/telefonicaid/fiware-orion/releases/tag/0.28.0:

> Fix: sanity check for string query 'q' - detect 'left-hand-side missing' (Issue #1754)

Thus, milestone has been changed accordingly.
